py-polars/tests/unit/functions/range/test_int_range.py

```diff
@@ -36,6 +29,13 @@ def test_int_range() -> None:
     expected = pl.Series("int_range", [0, 1, 2])
     assert_series_equal(pl.select(int_range=result).to_series(), expected)
 
+    # note: 'arange' is an alias for 'int_range'
+    ldf = pl.LazyFrame({"a": [1, 1, 1]})
+    assert_frame_equal(
+        ldf.filter(pl.col("a") >= pl.arange(0, 3)).collect(),
+        pl.DataFrame({"a": [1, 1]}),
+    )
```

Comment:
Was a slightly lazy avoidance of a false-positive from "typos"; shame on me ;p

Reply:
I'd think with the ignore in the typos config it would leave the test name alone :thinking: I guess it's fine with the new name as well.

Reply:
> I'd think with the ignore in the typos config it would leave the test name alone 🤔 I guess it's fine with the new name as well.

I expected the same, but something about it continued to trigger 😢